### PR TITLE
Fixes #16354 - Add rubocop

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: hooks/pre/30-upgrade.rb [hunk 1]
```diff
@@ -72,7 +72,7 @@ def fix_katello_settings_file
 
   return true unless settings.key?(:common)
 
-  settings = {:katello => settings[:common]}
+  settings = { :katello => settings[:common] }
```
Comment (ehelms) [inline hooks/pre/30-upgrade.rb]: Is this normal? Seems strange to have a leading space.

**Comment by ehelms** [comment]
Nice! I'd be thinking about this but of course you are the king of rubocop.

File: hooks/pre/30-upgrade.rb (hunk 1 above)
Comment (daviddavis) [inline hooks/pre/30-upgrade.rb]: That is odd but it seems that's [what rubocop does by default](https://git.io/v6x9L). I'm going to copy over the setting from katello: https://git.io/v6x9s

**Comment by daviddavis** [comment]
This should be good to go 👑

**Comment by ehelms** [comment]
ACK - thanks @daviddavis

**Pull request merged**